ci/setup_dcos-cli.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail
+
+if [ -z "${CLUSTER_URL}" ]; then
+    echo "Error: DC/OS cluster cluster url is not specified. Please, ensure CLUSTER_URL variable is exported in shell environment."
+    exit 1
+fi
+
+if [ $# -lt 2 ]; then
+    echo "Error: DC/OS cluster username or password is not specified. Please, ensure you run the script with paramenters
+     'setup_dcos-cli.sh <username> <password>'"
```

Comment:
Please be consistent with either underscores or dashes in file names.

Reply:
It is intentionally, because `dcos-cli` is well-known name and shouldn't be delimited.

Reply:
Not sure I understand the argument. Will changing the name to `setup_dcos_cli.sh` have any effect in behavior? If not let's be consistent with file names IMO.

Reply:
@VladimirKravtsov if we were actually referencing a file/binary called `dcos-cli` (with a dash) somewhere, then keeping the dash would make sense. But AFAICT all we mean here is `DC/OS CLI`, so it does not matter whether we use a dash or an underscore, therefore keeping the delimiter consistent makes more sense.